icons/premium-content-user-credit-24-f.svg

```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M5 14a3 3 0 1 1 3-3 3 3 0 0 1-3 3zm3 0H2a2 2 0 0 0-2 2v5a1 1 0 0 0 1 1h1v-5h1v6.428a.572.572 0 0 0 .572.572h2.856A.572.572 0 0 0 7 23.428V17h1v5h1a1 1 0 0 0 1-1v-5a2 2 0 0 0-2-2zM5 7a4.005 4.005 0 0 1 4 4 3.957 3.957 0 0 1-.58 2.042A2.995 2.995 0 0 1 11 16h.295A11.559 11.559 0 0 1 13 13.11V1.615c1.587 1.11 4.65 2.9 7.908 2.9A8.382 8.382 0 0 0 22 4.445v.136l-.025.63a32.694 32.694 0 0 1-.766 6.576l-.016-.007a7.106 7.106 0 0 1 .911.563A35.716 35.716 0 0 0 23 4.58V3.206C18.05 4.748 12.524 0 12.524 0L12.5.019 12.476 0S6.95 4.748 2 3.206V4.58c.04.894.063 2.136.195 3.575A3.985 3.985 0 0 1 5 7zm13 7a4 4 0 1 0 4 4 4.004 4.004 0 0 0-4-4zm6 4a6 6 0 1 1-6-6 6 6 0 0 1 6 6zm-1 0a5 5 0 1 0-5 5 5.006 5.006 0 0 0 5-5zm-11.697 2H11v1a1.997 1.997 0 0 1-.913 1.675A18.784 18.784 0 0 0 12.5 24c.386-.177.753-.363 1.111-.555A7.02 7.02 0 0 1 11.303 20z"/></svg>
\ No newline at end of file
+<svg id="icons" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M5 14a3 3 0 1 1 3-3 3 3 0 0 1-3 3zm3 0H2a2 2 0 0 0-2 2v5a1 1 0 0 0 1 1h1v-5h1v6.428a.572.572 0 0 0 .572.572h2.856A.572.572 0 0 0 7 23.428V17h1v5h1a1 1 0 0 0 1-1v-5a2 2 0 0 0-2-2zM5 7a4.005 4.005 0 0 1 4 4 3.957 3.957 0 0 1-.58 2.042A2.995 2.995 0 0 1 11 16h1.341A6.646 6.646 0 0 1 13 14.767V1.614c1.587 1.11 4.65 2.9 7.908 2.9A8.382 8.382 0 0 0 22 4.445v.136l-.025.63a31.446 31.446 0 0 1-.92 7.152 6.633 6.633 0 0 1 .906.47A34.533 34.533 0 0 0 23 4.58V3.206C18.05 4.748 12.524 0 12.524 0L12.5.019 12.476 0S6.95 4.748 2 3.206V4.58c.04.894.063 2.136.195 3.575A3.985 3.985 0 0 1 5 7zm6 13v1a1.997 1.997 0 0 1-.913 1.675A18.784 18.784 0 0 0 12.5 24c.49-.224.946-.467 1.395-.713A6.643 6.643 0 0 1 12.027 20zm7.5-4a2.5 2.5 0 1 0 2.5 2.5 2.5 2.5 0 0 0-2.5-2.5zm5.45 2.5a5.45 5.45 0 1 1-5.45-5.45 5.457 5.457 0 0 1 5.45 5.45zM22.1 18h.85a4.475 4.475 0 0 0-8.9 0h.85a3.61 3.61 0 0 0-.05.5 3.65 3.65 0 0 0 7.3 0 3.61 3.61 0 0 0-.05-.5z"/></svg>
```

Comment:
you have an id here, still, might be good to remove that to keep it consistent with the others.